openjdk/build.gradle

```diff
@@ -268,6 +268,22 @@ def addNativeTest(nativeClassifier) {
             executable = javaExecutable
             testClassesDir = testSourceSet.output.classesDir
 
+            // TODO(nmittler): Is there a way to copy all properties of the test task?
```

Comment:
It looks like you should be able to call InvokerHelper.setProperties(testLogging, test.testLogging.properties), but I don't know if that'll work directly in gradle files.